homeassistant/components/automate/sensor.py

```diff
@@ -40,8 +44,8 @@ def async_add_automate_sensors():
 class AutomateBattery(AutomateBase):
     """Representation of a Automate cover battery sensor."""
 
-    device_class = DEVICE_CLASS_BATTERY
-    unit_of_measurement = "%"
+    _attr_device_class = DEVICE_CLASS_BATTERY
+    _attr_unit_of_measurement = "%"
```

Comment:
Please use our common constants as much as possible. Here we should use this one:
https://github.com/home-assistant/core/blob/8d652b28e2790be695df154589d7a41cf30515fd/homeassistant/const.py#L505

Reply:
Funny you say that, when I started doing this (nearly a year ago), I was using them - but they got removed from `const.py` somewhere along the way and I needed to [stop using them](https://github.com/home-assistant/core/pull/39501/commits/a4a8ed331546b8ca6c66706591b3b5a93c2d16d6). I'll put them back as they have hopefully settled down a bit now.